### PR TITLE
docs: add example for string representation

### LINKED ISSUE #549: String representation of `bun.SelectQuery`
I cannot find on docs and on this Github repo a way to have the query's string representation.

I need to test that many cases (really many because we are exposing columns with possible `OR`, `AND` and `NOT` logical operators) are correctly generating SQL queries.

So I think to add many tests to check if the methods we have generate correct final SQL code.

Is there a way to do this?

If I had a way to generate the final SQL string I could just compare them in tests. End.

I searched the old `go-pg` repository and it seems possible to do this: https://github.com/go-pg/pg/issues/1366#issuecomment-659263194.

### PR DESCRIPTION
**Pull request by sashamelentyev**
Closes #549

### CODE REVIEW

File: example/string-representation/main.go [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	defer db.Close()
+
+	if err := createSchema(ctx, db); err != nil {
+		panic(err)
+	}
+
+	q := db.NewSelect().Model((*Item)(nil)).Where("id > ?", 0)
+
+	var buf []byte
+	buf, err = q.AppendQuery(schema.NewFormatter(sqlitedialect.New()), nil)
```
Comment (vmihailenco) [inline example/string-representation/main.go:36]: Please replace `schema.NewFormatter(sqlitedialect.New())` with `db.Formatter()`.

File: example/string-representation/main.go [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	if err != nil {
+		panic(err)
+	}
+
+	fmt.Println(string(buf))
+}
+
+func createSchema(ctx context.Context, db *bun.DB) error {
```
Comment (vmihailenco) [inline example/string-representation/main.go:44]: This can be removed, right?

File: example/string-representation/main.go (hunk 1 above)
Comment (sashamelentyev) [reply example/string-representation/main.go]: Done

File: example/string-representation/main.go (hunk 2 above)
Comment (sashamelentyev) [reply example/string-representation/main.go]: Yes. I removed

**Comment by vmihailenco** [comment]
Thanks! Perhaps as a next step we could also add `q.String()` that uses `q.AppendQuery(db.Formatter(), nil)` internally.

**Pull request merged**